src/cloudWatchLogs/activation.ts

```diff
@@ -15,27 +15,28 @@ import { LogStreamCodeLensProvider } from './document/logStreamCodeLensProvider'
 import { LogStreamDocumentProvider } from './document/logStreamDocumentProvider'
 import { LogGroupNode } from './explorer/logGroupNode'
 import { LogStreamRegistry } from './registry/logStreamRegistry'
+import { ExtContext } from '../shared/extensions'
 
-export async function activate(context: vscode.ExtensionContext, configuration: Settings): Promise<void> {
+export async function activate(extContext: ExtContext, configuration: Settings): Promise<void> {
```

Comment:
this change looks unnecessary. If the caller has ExtContext it can pass ExtContext.extensionContext

Reply:
I agree. This is a change that will be needed later in order to access the `guessDefaultRegion` method within the CloudWatchLogs Code by passing down the `AwsContext` object, but for now it seems best to leave it out.